tools/src/launcher/Main.hs

```diff
@@ -589,13 +586,8 @@ writeWindowsUpdaterRunner :: FilePath -> M ()
 writeWindowsUpdaterRunner runnerPath = liftIO $ do
     exePath <- getExecutablePath
     launcherArgs <- getArgs
-#ifdef mingw32_HOST_OS
-    selfPid <- Process.getCurrentProcessId
-#else
-    let selfPid = 0 -- This will never be run on non-Windows
-#endif
     writeFile (toString runnerPath) $ unlines
-        [ "TaskKill /PID "<>show selfPid<>" /F"
+        [ "TaskKill /IM cardano-launcher.exe /F"
         -- Run updater
```

Comment:
@KtorZ, if we merge this, `develop` will regress, since it's undoing the fix for https://iohk.myjetbrains.com/youtrack/issue/RCD-52